data/projects/editor_support.md

```diff
@@ -13,6 +13,28 @@ plugins. For some editors like sublime and gedit, running tests is not easy and
 there is no api to do it. In such cases we  would want to use dogtail
 (a GUI tester) or a similar tool to do this.
 
+Other details (for testing): 
+
+For sublime, we'll be using the unittest plugin which is a plugin by itself. 
+For atom, we have `apm test`.
+For VI and emacs, we will most likely use a testing framework such as Vimcasts.
+
+Other details (for maintenance):
+
+We will have to explore the possibility of having a single git repository for 
+atom, sublime-text and vim as it will ultimately be easier to maintain in the
+long run and will have a proper CI, which can be easily managed. Issues will 
+be raised more frequently as a result of this.
```

Comment:
@Udayan12167 
The motivation behind this was that the editor plugins don't get enough attention. Having a unified repo which we might be able to do for the above mentioned plugins, and have separate repos for the rest may be a potential solution as more users will be bound to be using at least one of the plugins in the unified repo, hence if something breaks; an issue will be raised, it'll be easier to deal with since the tests on that repo will fail and we wont have to go through all of the editors individually. We'd have to weigh-in the drawbacks, as it'd involve a tradeoff for certain plugins wherein installation is easier if it's in a separate repo. 
eg. vim: `PluginInstall <repo>`
I asked a few maintainers and potential mentors, with varied opinions on this. It's worth a try, but it might not be a priority to the project.